frontend/src/store/ApplicationState.ts

```diff
@@ -37,3 +39,12 @@ export const selectedApplicationDisplayText = computed(() => {
         return '';
     }
 });
+
+export const useNotificationMessage = reactive({
+    notificationMsg: '',
+    isNotificationVisible: false,
+})
+
+export const useErrorDialog = reactive({
+    isErrorVisible: false,
+})
```

Comment:
Hmm... I am confused why "grantAccessFormData", "useNotificationMessage" and "useErrorDialog" belong to this "ApplicationState.ts" file as originally it seems to be holding state for "Application" but Notification and Dialog different. Can they be in different "state" file or is it better to design/rename the filename to be more generic?